execution/executor-service/src/remote_cordinator_client.rs

```diff
@@ -78,11 +80,26 @@ impl CoordinatorClient<RemoteStateViewClient> for RemoteCoordinatorClient {
     fn receive_execute_command(&self) -> ExecutorShardCommand<RemoteStateViewClient> {
         match self.command_rx.recv() {
             Ok(message) => {
-                let request: RemoteExecutionRequest = bcs::from_bytes(&message.data).unwrap();
+                let _rx_timer = REMOTE_EXECUTOR_TIMER
+                    .with_label_values(&[&self.shard_id.to_string(), "cmd_rx"])
+                    .start_timer();
+                let request: RemoteExecutionRequest;
+                {
+                    let _bcs_deser_timer = REMOTE_EXECUTOR_TIMER
```

Comment:
Hmm I don't like this style of dropping the timer - it is confusing... may be less confusing to remove the implicit scope and drop the timer instead..?

Reply:
You mean, I should use stop_and_discard() ?

Reply:
I meant to do something like this ..

```
let bcs_deser_timer = REMOTE_EXECUTOR_TIMER.with_label_values(&[&self.shard_id.to_string(), "cmd_rx_bcs_deser"])
                        .start_timer();
let request = bcs::from_bytes(&message.data).unwrap();
drop(bcs_deser_timer)
```

Reply:
Here and in other places as well..